scripts/qa_crowbarsetup.sh

```diff
@@ -563,6 +563,12 @@ function get_unclustered_sles12plus_nodes
     echo $sles12plusnodes
 }
 
+function get_non_controller_nodes
+{
+    sles12nodes=get_unclustered_sles12plus_nodes
+    controller_node=get_novacontroller
+    echo ${sles12nodes[@]} ${controller_node[@]} | tr ' ' '\n' | sort | uniq -u
```

Comment:
This function is supposed to output non-controller nodes, right? So I'm confused why it is outputting controller nodes here.